doc/source/intro.rst

```diff
@@ -4,17 +4,24 @@ Getting started
 Installation
 ^^^^^^^^^^^^
 
-PyDPF Composites supports Ansys version 2023 R1 and later. Make sure you have a licensed copy of Ansys installed.
+PyDPF Composites supports Ansys version 2023 R1 and later. Make sure you have a licensed copy of Ansys installed. See
+:ref:`Compatibility` to understand which ansys-dpf-composites version corresponds to which Ansys version.
 
 Install the ``ansys-dpf-composites`` package with ``pip``:
 
 .. code::
 
     pip install ansys-dpf-composites
 
+Specific versions can be installed by specifying the version in the pip command. For example: Ansys 2023 R1 requires ansys-dpf-composites version 0.1.0:
+
+.. code::
+
+    pip install ansys-dpf-composites==0.1.0
 
-You should use a `virtual environment <https://docs.python.org/3/library/venv.html>`_,
-because it keeps Python packages isolated from your system Python.
+
+It is recommended to use a `virtual environment <https://docs.python.org/3/library/venv.html>`_,
```

Comment:
The previous language here was introduced in the overall doc edit (https://github.com/ansys/pydpf-composites/pull/186). I think we should keep it.